src/EnergyPlus/WeatherManager.cc

```diff
@@ -2558,7 +2558,7 @@ namespace Weather {
                     } else { // ReadStatus /=0
                         if (WeatherDataLine.eof &&
                             state.dataWeather->NumDataPeriods == 1) { // Standard End-of-file, rewind and position to first day...
-                            if (state.dataWeather->DataPeriods(1).NumDays >= state.dataWeather->NumDaysInYear) {
+                            if (state.dataWeather->DataPeriods(1).NumDays >= Weather::NumDaysInYear) {
```

Comment:
Was there a reason for just this one change in WeatherManager?

Reply:
When I changed it to initialize in the declaration `int const NumDaysInYear=365` one of the custom checks complained that this should be `constexpr` which then meant it had to move out of `state`.